dd-sdk-android-gradle-plugin/src/main/kotlin/com/datadog/gradle/plugin/internal/GitRepositoryDetector.kt

```diff
@@ -74,4 +100,11 @@ internal class GitRepositoryDetector : RepositoryDetector {
     }
 
     // endregion
+
+    companion object {
+        const val WRONG_URL_FORMAT_WARNING_MESSAGE =
+            "The detected GIT remote url: [%s] is not of 'http[s]' and it will" +
+                "not work properly with this plugin. You can provide a different URL through" +
```

Comment:
just a thought: user may still wonder what will not work properly, maybe we can just say it won't be browsable from dashboard? or make a link to the feature. wdyt?